modules/currency-l1/src/main/scala/org/tessellation/currency/l1/CurrencyL1App.scala

```diff
@@ -1,13 +1,13 @@
 package org.tessellation.currency.l1
 
 import cats.effect.{IO, Resource}
+import cats.implicits.catsSyntaxOptionId
```

Comment:
please don't mix `cats.implicits` with `cats.syntax`. you can import `cats.syntax.option._` or just remove all and import `cats.syntax.all._`